### DISCUSSION TITLE
鸭脖事件

### DISCUSSION THREAD

**Discussion by vaynevayne** [epicmaxco/vuestic-ui#3468] [Q&A]
Excuse me, is there any drama in your country?
I want to escape, but I don't have any skills other than writing code.

**Comment by asvae** [comment]
You mean like theatre?

Coding skill are pretty good fit for nomadic life, so why not?

**Reply by vaynevayne** [reply]
Brother, do you remember me? It's so hard, I lost my job
The domestic interview is all about whether there is any experience in a big factory. I used to have the opportunity to go to a big company, but I didn't go. I wanted to make some achievements in a small company, but I was laid off. I didn't want to find a good job because I didn't participate in open source. I was too unwilling. am I afraid to go abroad? I am a road addict
I don't have any friends. I'm just telling you the whole thing. @asvae
